### ISSUE TITLE
Signal - File - Error org.asamk.Signal.Error.Failure: Failure: [404] Not found (NotFoundException)

### ISSUE THREAD

**Issue by carolinaxxxxx** [agileek/hassio-addons#84]
https://github.com/AsamK/signal-cli/issues/1609

**Comment by carolinaxxxxx** [comment]
@MichaelBitard will you help? Thx.

**Comment by carolinaxxxxx** [comment]
@tholland15 @marecabo @JayNewstrom anywon?

**Comment by tholland15** [comment]
@carolinaxxxxx I was also hit by this a few weeks ago but there were no issues I could find about it at the time. Messages that included images had been silently failing to send for weeks before I noticed. 

I moved over to the semi-official Signal Messenger Rest API integration: https://www.home-assistant.io/integrations/signal_messenger/ which was a fairly easy transition and has been 100% stable since. I just had to update my service calls for those that sent images in the messages since the yaml for this integration's service calls is a slightly different format when sending images (iirc it was exactly the same when sending text only)
